server/routes.go

```diff
@@ -24,7 +24,7 @@ func DoRoutes() chi.Router {
 	r.Route("/api/entitlements/v1", func(r chi.Router) {
 		r.With(identity.EnforceIdentity).Route("/", controllers.LubDub)
 		r.Route("/openapi.json", apispec.OpenAPISpec)
-		r.With(identity.EnforceIdentity).Get("/services", controllers.Index(nil))
+		r.With(identity.EnforceIdentity).Get("/services", controllers.Index(nil, nil))
```

Comment:
oh man, this is super weird to have a `func` being called with `nil` and `nil`. we should think about how this can be re-written so that it's easy to grok and still easy to test.

Reply:
@shdunning  Yeah, same reason as above! Any ideas?

Reply:
@shdunning I can probably have a function that calls both getSubscriptions and bundleInfo, but that might overcomplicate things

Reply:
So, a couple of approaches in order from low effort to high effort:

1. Since both of your composable functions are `var` declarations in the `controllers` package, you could (and probably should do this anyway) make `controllers_test.go` part of the same `controllers` package and then you can reference those vars directly in your unittests. Then there would be no need to have `Index` take the 2 `func`s as arguments, and it would clean up the code a bit. For example, there would no longer be a need for things like:
```
		if getCall == nil {
			getCall = getSubscriptions
		}

		if getBundleInfo == nil {
			getBundleInfo = BundleInfo
		}
```
2. The more interesting question is why aren't `getBundleInfo` and `getSubscriptions` testable themselves with the right set of mocks in place? `getSubscriptions` should be mockable using recorders via `httptest` and we should be able to find a way mock up `	yamlFilePath := config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml)` to point to a test yaml file. I _think_ the reason why this wasn't working before is b/c `controllers.go` and `controllers_test.go` aren't part of the same package (should both be `controllers`). 

I think item #1 above gets us the most bang for our buck -- low effort and some clean up -- but we should queue up some tech debt to address #2 as it will have higher return in writing cleaner / better Go code :-)